index.bs

```diff
@@ -192,6 +192,7 @@ interface ImageCapture {
     readonly attribute MediaSettingsRange         imageHeight;
     readonly attribute MediaSettingsRange         imageWidth;
     readonly attribute FrozenArray&lt;FillLightMode> fillLightMode;
+    readonly attribute FrozenArray&lt;DOMString>     supportedImageFormats;
```

Comment:
Hmm don't fully grasp the question, could you please rewrite it or provide an example...?